### ISSUE TITLE
MLXX NB4 MODELLING.ipynb

### ISSUE THREAD

**Issue by mivige** [mivige/NataVisionaries#5]
- [x] Train/test split
- [x] Experiment with ML classifiers (e.g. RF, XGBoost, etc.)
- [x] Compare accuracy
- [x] Optional hyperparameter search
